README.rst

```diff
@@ -79,6 +79,17 @@ Here at SparkPost, our messages are known as transmissions. Let's use the underl
 
 .. _transmissions API: https://www.sparkpost.com/api#/reference/transmissions
 
+Django Integration
+------------------
+SparkPost python library comes with an email backend for Django. So, all you need is to use this backend in Django and put the API Key.
```

Comment:
Couple copy edits:

- "The" SparkPost python library...
- Delete the second sentence.

Net is this:

`The SparkPost python library comes with an email backend for Django. Put the following configuration in your settings.py file:`